src/Chat.tsx

```diff
@@ -113,6 +116,17 @@ export class Chat extends React.Component<ChatProps, {}> {
         });
     }
 
+    private handleKeyDown(evt: any) {
+        // Chat textbox has "onKeyDown" handler that will call evt.stopPropagation, preventing the handled event to bubble up to this point
+        if (
```

Comment:
Remember that Adaptive Cards may also have input elements (like textboxes, checkboxes, etc.) These may not have stopPropagation handlers on them. Can you test with a card like [this](http://adaptivecards.io/samples/#InputForm)?

Reply:
Tested with Adaptive Cards.